controls/cis_rhel10.yml

```diff
@@ -1047,7 +1047,7 @@ controls:
       - l1_workstation
     status: automated
     notes: Review the availability of this package when the product is out.
-    rules:
+    related_rules:
```

Comment:
Could you also remove the comment I put in the notes, please? : )

Reply:
Done.